www/util.php

```diff
@@ -1398,8 +1386,7 @@ function summarizeHtml($str, $maxlen)
                 if (strncasecmp($entTxt, "lt;", 3) == 0
```

Comment:
a line above here is `$entTxt = substr($str, $ofs + 1, 7);`
I think the 7 can be reduced to 5, which is the farthest ahead this little test cares about.

It was 7 in the original code import, when the regex was `{1,5}`. When I changed the regex to `{1,7}`, I should have increased that number to 9. It's nice to find bugs after they're obsolete.